packages/rs-drive/src/drive/identity/withdrawals/transaction_index_counter.rs

```diff
@@ -0,0 +1,205 @@
+use grovedb::{
+    query_result_type::{QueryResultElement, QueryResultType},
+    Element, PathQuery, Query, SizedQuery, TransactionArg,
+};
+
+use crate::{
+    drive::{
+        batch::{drive_op_batch::WithdrawalOperationType, DriveOperationType},
+        Drive, RootTree,
+    },
+    error::{drive::DriveError, Error},
+};
+
+use super::paths::{
+    get_withdrawal_transactions_expired_ids_path,
+    get_withdrawal_transactions_expired_ids_path_as_u8, WITHDRAWAL_TRANSACTIONS_COUNTER_ID,
+};
+
+impl Drive {
+    /// Get latest withdrawal index in a queue
+    pub fn fetch_latest_withdrawal_transaction_index(
```

Comment:
This function name isn't expressing that it's deleting from the backing store. It's a very bad practice to have such hidden code and this can lead to unknown bugs. Rename to remove_latest_withdrawal_transaction_index